contracts/OtcEscrowApprovals.sol

```diff
@@ -43,11 +47,17 @@ contract OtcEscrowApprovals {
 
         balAmount = balAmount_;
         aaveAmount = aaveAmount_;
+
+        hasSwapOccured = false;
```

Comment:
No need for this since it will default to false

Reply:
Done